### DISCUSSION TITLE
Run fastlane ios_build, it ask me to select which target should choose

### DISCUSSION THREAD

**Discussion by walker2021wyze** [fastlane/fastlane#20037] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/86333976/157026647-1c75e55b-e1db-4c97-aa82-749824c1bc90.png)

**Comment by walker2021wyze** [comment, marked as answer]
this is the lane of  wyze_build

    gym(
       workspace:"Wyze.xcworkspace",
       scheme:"Wyze", 
       target:"Wyze", 
       export_method:"enterprise",
       configuration:"ReleaseEnterpriseCN",
       output_name:"#{ipaName}",
       output_directory:"./build",
       export_options:"../.github/scripts/ExportOptions.plist"
     )

**Comment by gabimoncha** [comment]
@walker2021wyze I'm getting this error:
```
Could not find option 'target' in the list of available options
```
